src/fidesops/service/connectors/saas_query_config.py

```diff
@@ -80,7 +81,12 @@ def generate_query(
                 path = path.replace(f"<{param.name}>", input_data[param.name][0])
 
         logger.info(f"Populated request params for {current_request.path}")
-        return "GET", path, params, None
+        method: HTTPMethod = (
+            current_request.method if current_request.method else HTTPMethod.GET
```

Comment:
Here and below feel like they're places too nested within the traversal to be setting defaults.

I don't think the request's method is ever really optional, so we should probably force the specification of it every time to be explicit and standard across the language.

Reply:
it is optional in the sense that we have defaults if not specified: `GET`  for `read` and `PUT` for `update` requests

Reply:
should we change this to be explicit across all configs instead?

Reply:
this will be follow-up ticket

Reply:
https://github.com/ethyca/fidesops/issues/305